lazer/evm/src/ExampleReceiver.sol

```diff
@@ -7,15 +7,22 @@ import {PythLazerLib} from "pyth-lazer/PythLazerLib.sol";
 
 contract ExampleReceiver {
     PythLazer pythLazer;
+    uint256 verification_fee;
     uint64 public price;
     uint64 public timestamp;
 
     constructor(address pythLazerAddress) {
         pythLazer = PythLazer(pythLazerAddress);
+        verification_fee = pythLazer.verification_fee();
```

Comment:
I am storing the fee since making read calls for it is a waste of gas.

Reply:
I think we shouldn't do that in the example, otherwise the consumers will just copy that. We may change the verification fee at any time and that shouldn't break consumer contracts.